src/main/java/spotify/api/enums/AuthorizationScope.java

```diff
@@ -23,6 +34,11 @@
 
     private final String value;
 
+    /**
+     * Set the authorization scope.
+     * 
```

Comment:
Same feedback as with the javadoc for the `AlbumType` constructor. So I would like to refer you to that.